drivers/gles3/shader_gles3.cpp

```diff
@@ -250,7 +250,6 @@ ShaderGLES3::Version *ShaderGLES3::get_current_version() {
 		ERR_FAIL_COND_V(!custom_code_map.has(conditional_version.code_version), NULL);
 		cc = &custom_code_map[conditional_version.code_version];
 		v.code_version = cc->version;
-		define_line_ofs += 2;
```

Comment:
Same as above, if some code were added after this statement and used `define_line_ofs`, it would be quite easy to forget to add this increment here, and thus introduce bugs.